chromium/popup.css

```diff
@@ -1,132 +1,115 @@
+  margin-top: 0;
   margin-bottom: 0;
   min-width: 20em;
+  font-family: 'Lucida Grande', 'Segoe UI', Tahoma, 'DejaVu Sans', Arial, sans-serif;
+  font-size: 75%;
+  color: #303942;
+}
+
+p {
+  line-height: 1.8em;
+}
+
+h1, h2, h3 {
+  user-select: none;
+  font-weight: normal;
+  line-height: 1;
+}
+
+h1 {
+  font-size: 1.5em;
+}
+
+h2 {
+  font-size: 1.3em;
+  margin-bottom: 0.4em;
+}
+
+h3 {
+  color: black;
+  font-size: 1.2em;
+  margin-bottom: 0.8em;
+}
+
+a {
+  color: #15c;
+  text-decoration: underline;
+}
+
+a:active {
+  color: #052577;
 }
 
-/* Don't wrap text for important stuff. */
 h1, h2, h3, .rule {
   white-space: nowrap;
 }
 
-/* Hide rules & options if the extension is off. */
-.disabled #RuleManagement, .disabled #HttpNowhere {
+.disabled #RuleManagement,.disabled #HttpNowhere {
   display: none;
 }
 
-/* Initially hide section (until rules get added). */
 section.rules {
-  position: fixed;
-  visibility: hidden;
+  display: none;
 }
 
-.rule{
+.rule {
   height: 25px;
 }
 
-.rule label{
-  padding: 0px;
+.rule label {
+  padding: 0;
   max-width: 93%;
   overflow: hidden;
 }
 
-/* Underline rules that have notes. */
 .rule [title] {
   border-bottom: 1px dotted;
   cursor: help;
 }
 
-/* Favicons */
-.rule img.favicon {
-  width: 16px;
-  height: 16px;
-  margin-left: 0.6em;
-  vertical-align: bottom;
-}
-
-/* Remove user rule icon */
-.rule img.remove {
-  width: 15px;
-  height: 15px;
-  float: right;
-  opacity: .8;
-}
-
-.rule img.remove:hover {
-  opacity: 1;
-}
-
-/*** Everything below is "reverse-engineered" from Chrome's Settings page. ***/
-
-/* Override a strange pixel offset in widgets.css. */
-.rule input[type="checkbox"] {
-  bottom: 1px;
-  margin: 0px;
+.rule input[type='checkbox'] {
+  margin: 0;
 }
 
-/* Override display attribute of text labels in widgets.css. */
 .rule span {
-  display: inline !important;
   margin-left: 0.6em;
 }
 
-body {
-  /* Fonts Chrome specifies for each OS:     */
-  /* Linux: 'DejaVu Sans', Arial, sans-serif */
-  /* Mac: 'Lucida Grande', sans-serif        */
-  /* Windows: 'Segoe UI', Tahoma, sans-serif */
-  font-family: 'Lucida Grande', 'Segoe UI', Tahoma, 'DejaVu Sans', Arial, sans-serif;
-  font-size: 75%;
-  color: #303942;
-  padding-top: 55px;
+.rule img.remove {
+  float: right;
 }
 
 header {
-  background-image: linear-gradient(white, white 40%, rgba(255, 255, 255, 0.92));
-  left: 1em;
-  position: fixed;
-  right: 1em;
-  top: 0;
-  z-index: 3;
+  border-bottom: 1px solid #eee;
 }
 
 header > h1 {
   margin: 0;
   padding: 21px 0 13px;
 }
 
-/* Create a border under the h1 (but before anything that gets appended
- * to the end of the header). */
-header > h1::after {
-  background-color: #eee;
-  content: ' ';
-  display: block;
-  height: 1px;
-  position: relative;
-  top: 13px;
-}
-
 section {
-  -webkit-padding-start: 18px;
-  -webkit-padding-end: 18px;
-  margin-bottom: 24px;
-  margin-top: 8px;
+  padding-left: 18px;
+  padding-right: 18px;
```

Comment:
I think the padding-left (indentation) is fine, but the right-hand padding removes a lot of text rendering area, pushing content to the next line.  I'd remove this line.